lib/modules/manager/bazel-module/extract.spec.ts

```diff
@@ -618,5 +618,375 @@ describe('modules/manager/bazel-module/extract', () => {
+            currentValue: '1.2.1',
+          },
+          {
+            datasource: BazelDatasource.id,
+            depType: 'bazel_dep',
+            depName: 'platforms',
+            currentValue: '0.0.8',
+          },
+          {
+            datasource: BazelDatasource.id,
+            depType: 'bazel_dep',
+            depName: 'rules_img',
+            currentValue: '0.1.5',
+          },
+          {
+            datasource: BazelDatasource.id,
+            depType: 'bazel_dep',
+            depName: 'stardoc',
+            currentValue: '0.6.2',
+          },
+          {
+            datasource: DockerDatasource.id,
+            depType: 'rules_img_pull',
+            depName: 'ubuntu',
+            packageName: 'index.docker.io/library/ubuntu',
+            currentValue: '24.04',
+            currentDigest:
+              'sha256:1e622c5f9ac0c0144d577702ba5f2cce79fc8e3cf89ec88291739cd4eee3b7b9',
+            replaceString: codeBlock`
+              pull(
+                  name = "ubuntu",
+                  digest = "sha256:1e622c5f9ac0c0144d577702ba5f2cce79fc8e3cf89ec88291739cd4eee3b7b9",
+                  registry = "index.docker.io",
+                  repository = "library/ubuntu",
+                  tag = "24.04",
+              )
+            `,
+          },
+        ],
+      });
+    });
+
+    it('returns rules_img pull dependencies', async () => {
+      const input = codeBlock`
+        bazel_dep(name = "rules_img", version = "0.1.0")
+        pull = use_repo_rule("@rules_img//img:pull.bzl", "pull")
+        pull(
+            name = "ubuntu",
+            digest = "sha256:1e622c5f9ac0c0144d577702ba5f2cce79fc8e3cf89ec88291739cd4eee3b7b9",
+            registry = "index.docker.io",
+            repository = "library/ubuntu",
+            tag = "24.04",
+        )
+      `;
+
+      const result = await extractPackageFile(input, 'MODULE.bazel');
+
+      expect(result).toEqual({
+        deps: [
+          {
+            datasource: BazelDatasource.id,
+            depType: 'bazel_dep',
+            depName: 'rules_img',
+            currentValue: '0.1.0',
+          },
+          {
+            datasource: DockerDatasource.id,
+            depType: 'rules_img_pull',
+            depName: 'ubuntu',
+            packageName: 'index.docker.io/library/ubuntu',
+            currentValue: '24.04',
+            currentDigest:
+              'sha256:1e622c5f9ac0c0144d577702ba5f2cce79fc8e3cf89ec88291739cd4eee3b7b9',
+            replaceString: codeBlock`
+              pull(
+                  name = "ubuntu",
+                  digest = "sha256:1e622c5f9ac0c0144d577702ba5f2cce79fc8e3cf89ec88291739cd4eee3b7b9",
+                  registry = "index.docker.io",
+                  repository = "library/ubuntu",
+                  tag = "24.04",
+              )
+            `,
+          },
+        ],
+      });
+    });
+
+    it('returns rules_img pull dependencies with custom registry', async () => {
```

Comment:
If custom registry is detected, it should be set in the registryUrl field

Reply:
This is now implemented with the latest commit.